HOWTO.md

```diff
@@ -21,24 +21,46 @@ It can be controlled in a variety of ways:
 
 ## Preflight requests
 
-PNA preflight requests are available in Chrome starting in version 98 behind
-a pair of command-line flags:
+PNA preflight requests are available in Chrome starting in version 98 under warning-mode.
+It can be switched between enforcing-mode and warning-mode in following ways:
 
-1. `--enable-features=PrivateNetworkAccessSendPreflights`
-2. `--enable-features=PrivateNetworkAccessRespectPreflightResults`
+* the command-line flags: `--enable-features=PrivateNetworkAccessRespectPreflightResults`
+* the `chrome://` flag: `chrome://flags/#private-network-access-respect-preflight-results`
 
-The former configures Chrome to send preflight requests ahead of private network
-requests, but not does require the response to be successful. The latter enables
-enforcement: if the preflight request fails, then the request is not sent.
+Chrome is sending preflight requests ahead of private network requests, but does 
+not require the response to be successful. The flag above enables enforcement: 
+if the preflight request fails, then the request is not sent.
 
-See [here](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)
-for instructions on setting command-line flags.
+### Navigations and Workers
 
-You can then test your own web app, or use a test page like
-https://private-network-access-test.glitch.me/.
+PNA preflight requests are also available for navigations and workers under the 
+following command-line flags:
+* `--enable-features=PrivateNetworkAccessForNavigations`
+* `--enable-features=PrivateNetworkAccessForWorkers`
+* `--enable-features=PrivateNetworkAccessForNavigationsWarningOnly`
+* `--enable-features=PrivateNetworkAccessForWorkersWarningOnly`
 
-Errors should manifest as CORS errors in the DevTools console, and as "Blocked
-requests" in the DevTools network panel.
+The first two flags enable the features under warning-mode. The second two flags
+works only when the first two are enabled and enables the enforcement of the 
+features.
```

Comment:
It's actually the opposite.  The first two flags enable the enforcing mode, unless the latter two flags are also enabled, then the warning mode is on.